src/robusta/utils/service_discovery.py

```diff
@@ -18,6 +18,6 @@ def find_service_url(label_selector):
     name = svc.metadata.name
     namespace = svc.metadata.namespace
     port = svc.spec.ports[0].port
-    url = f"http://{name}.{namespace}.svc.cluster.local:{port}"
+    url = f"http://{name}.{namespace}:{port}"
```

Comment:
Same here,
I would pass the helm value to the runner, using env variable, and use it here